[CCV] CUR 2.0 - update public docs following CUR changes (#20867)
DataDog/documentation: content/en/cloud_cost_management/aws.md

## Overview

> To use Cloud Cost Management, you must have an AWS account with access to Cost and Usage Reports (CURs), and have the AWS integration installed in Datadog. To set up Cloud Cost Management in Datadog, you need to generate a legacy Cost and Usage report.

Reviewer: ```suggestion
To use Cloud Cost Management, you must have an AWS account with access to Cost and Usage Reports (CURs), and have the AWS integration installed in Datadog. To set up Cloud Cost Management in Datadog, you need to generate a legacy Cost and Usage report through AWS Billing.
```

Reviewer: Targeting this: 
<img width="359" alt="image" src="https://github.com/DataDog/documentation/assets/17853509/3d52e962-0680-4858-bab2-16f91a55726e">

and not this:
<img width="1542" alt="image" src="https://github.com/DataDog/documentation/assets/17853509/24fc95dd-f8ea-4bf8-ae70-417e6c6f41be">

## Setup

> ### Prerequisite: generate a Legacy Cost and Usage Report

Reviewer 2: ```suggestion
### Prerequisite: generate a legacy Cost and Usage Report
```

## Overview

To set up Cloud Cost Management in Datadog, you should:

> 1. Have an AWS account with billing access
> 2. Have the AWS integration installed in Datadog
> 3. Follow the steps below to create a Cost and Usage report

Reviewer 3: ```suggestion
1. Have an AWS account with billing access.
2. Have the AWS integration installed in Datadog.
3. Follow the steps below to create a Cost and Usage report.
```

## Setup

### Prerequisite: generate a Cost and Usage Report

> [Create a Cost and Usage Report][1] in AWS under the **Legacy Pages** section. You cannot create a report with data exports.

Reviewer: ```suggestion
[Create a Cost and Usage Report][1] in AWS under the **Legacy Pages** section. At this time, there is no support for creating a Cost and Usage Report data exports.
```

> [Create a Cost and Usage Report][1] in AWS under the **Legacy Pages** section. At this time, there is no support for creating Cost and Usage Report data exports.

Author: AWS released last week a new way of generating billing exports called "data exports". The previous way that we support has been moved under the "Legacy Pages" section.
We want to make it clear that they should continue to use the legacy way. We plan to provide support for data exports in the near future.

> [Create a Cost and Usage Report][1] in AWS under the **Legacy Pages** section. At this time, there is no support for creating a Cost and Usage Report data exports.

Reviewer: ```suggestion
[Create a Cost and Usage Report][1] in AWS under the **Legacy Pages** section. At this time, there is no support for creating Cost and Usage Report data exports.
```

This ensures complete cost accuracy by allowing periodic cost calculations against the AWS Cost Explorer.

> ### Locate the Legacy Cost and Usage Report

Reviewer 2: ```suggestion
### Locate the legacy Cost and Usage Report
```

**Note**: Datadog only supports CURs generated by AWS. Do not modify or move the files generated by AWS, or attempt to provide access to files generated by a 3rd party.

> ### Configure access to the Legacy Cost and Usage Report

Reviewer 2: ```suggestion
### Configure access to the legacy Cost and Usage Report
```
